sound/soc/intel/boards/sof_sdw_rt_dmic.c

```diff
@@ -47,6 +51,17 @@ int rt_dmic_rtd_init(struct snd_soc_pcm_runtime *rtd)
 	if (!card->components)
 		return -ENOMEM;
 
+	codec_dai = get_codec_dai_by_name(rtd, multi_function_dmics,
+					  ARRAY_SIZE(multi_function_dmics));
+	if (codec_dai) {
+		/* Add -mic suffix for multi function dmics */
+		card->components = devm_kasprintf(card->dev, GFP_KERNEL,
+						  "%s-mic", card->components);
```

Comment:
so, this will result ` mic: rt722-sdca-mic` in the components list, right?
But since the `rt722-sdca` is after the `mic:` tag, it is a mic, why it is needed to be spelled out?
And the `spk:` is sans `-sdca`?